packages/harpguru-core/src/components/harp-row/utils/get-harp-cells/get-harp-cells.test.tsx

```diff
@@ -9,12 +9,23 @@ import { getHarpCells } from './get-harp-cells'
 
 jest.mock('reactn')
 const mockUseGlobal = useGlobal as jest.Mock
+const layoutFacts = {
```

Comment:
Already addressed in a code review above, but this comment needs removing and the code modifying here too.